sktime/forecasting/adapters/__init__.py

```diff
@@ -1,6 +1,10 @@
 """Module containing adapters to other forecasting framework packages."""
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 
-__all__ = ["HCrystalBallAdapter"]
+__all__ = [
+    "HCrystalBallAdapter",
+    "AutoTSAdapter",
```

Comment:
I would move it to `sktime.autots`, since this is not a full framework adapter

Reply:
Shouldn't it be `sktime.forecasters.autots`?

Reply:
yes, of course, apologies for the confusion.